en/config_mc/filter_tuning.md

```diff
@@ -27,7 +27,8 @@ The following factors affect control latency:
   4 kHz or higher is only recommended for controllers with STM32H7 processor or newer (2 kHz value is near the limit for less capable processors).
 - The IO chip (MAIN pins) adds about 5.4 ms latency compared to using the AUX pins (this does not apply to a *Pixracer* or *Omnibus F4*, but does apply to a Pixhawk).
   To avoid the IO delay, disable [SYS_USE_IO](../advanced_config/parameter_reference.md#SYS_USE_IO) and attach the motors to the AUX pins instead.
-- PWM output signal: enable [Dshot](../peripherals/dshot.md) or One-Shot ([PWM_AUX_RATE=0](../advanced_config/parameter_reference.md#PWM_AUX_RATE) or [PWM_MAIN_RATE=0](../advanced_config/parameter_reference.md#PWM_MAIN_RATE)) to reduce latency.
+- PWM output signal: enable [Dshot](../peripherals/dshot.md) or One-Shot to reduce latency.
+  The protocol is selected for a group of outputs during [Actuator Configuration](../config/actuators.md).
```

Comment:
This is the "standard text" I used everywhere for enabling the protocol.

Reply:
Makes sense.